File: Yndx_Algo_1/src/Lesson1/java/Task_2.java
package Lesson1.java;

import java.util.Scanner;

public class Task_2 {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();

        System.out.println(isTriangle(a,b,c));

    }

    public static String isTriangle(int a, int b, int c) {
        return (a + b > c && a + c > b && c + b > a) ? "YES" : "NO";
    }
}

File: Yndx_Algo_1/src/Lesson1/kotlin/Task_2.kt
package Lesson1.kotlin

/*Даны три натуральных числа. Возможно ли построить треугольник с такими сторонами. Если это возможно, выведите строку YES, иначе выведите строку NO.
Треугольник — это три точки, не лежащие на одной прямой.*/

fun main(args:Array<String>){

    var a = readLine()!!.toInt()
    var b = readLine()!!.toInt()
    var c = readLine()!!.toInt()

    if (a+b>c && a+c>b && b+c>a)
        println("YES")
    else
        println("NO")
}